shared/subspace-proof-of-space-gpu/README.md

```diff
@@ -3,6 +3,8 @@
 This crate exposes some low-level primitives to accelerate proof of space implementation on Nvidia (CUDA, Volta+) and
 AMD (ROCm) GPUs.
 
+The implementation expects 64KB of shared memory and about 1.1GB of VRAM on GPU.
```

Comment:
This seems very small?

```suggestion
The implementation expects 64MB of shared memory and about 1.1GB of VRAM on GPU.
```

Reply:
We are talking about low-level shared memory on GPU, probably something more like caches on CPU, so I don't think it is wrong